KatelynKim/RecipeChecklist
# Set up a db connection

### Summary 
Added `utils/dbConnect.ts` to make a connection to the Mongo DB cluster. 

Dependencies added:
  * mongoose ^6.6.7
  * types/mongoose ^5.11.97

References [RA-31](https://vandev.atlassian.net/browse/RA-31)

### Steps to test
1. Inside the `.env.local` file, enter the mongo urI (shared in discord #resources chat).
2. `npm run dev` and route to `http://localhost:3000/api/hello`
3. Ensure that you can see the sample data.
